apps/ledger-live-desktop/tests/models/DeviceAction.ts

```diff
@@ -3,6 +3,7 @@ import {
   deviceInfo155 as deviceInfo,
   mockListAppsResult as innerMockListAppResult,
 } from "@ledgerhq/live-common/apps/mock";
+import { DeviceModelId } from "@ledgerhq/devices";
```

Comment:
`DeviceModelId` has been put into the new typing system. It should be:
```
import { DeviceModelId } from "@ledgerhq/types-devices";
```
now ;)

Reply:
thanks!